Voycar.Api.Web/Features/Members/Get/Verify/Endpoint.cs

```diff
@@ -28,32 +28,32 @@ public override void Configure()
         this.AllowAnonymous();
         this.Summary(s =>
         {
-            s.Summary = "Verify Member";
-            s.Description = "Verify a member against the database " +
+            s.Summary = "Verify user";
+            s.Description = "Verify a user against the database " +
                             "and update last time of verification";
```

Comment:
das "last" kann raus
```suggestion
                            "and update time of verification";
```